lib/src/room.dart

```diff
@@ -265,10 +271,12 @@ class Room {
       return isDirectChat ? result : i18n.groupWith(result);
     }
     if (membership == Membership.invite) {
-      final sender = getState(EventTypes.RoomMember, client.userID!)
-          ?.senderFromMemoryOrFallback
-          .calcDisplayname(i18n: i18n);
-      if (sender != null) return sender;
+      final ownMember = unsafeGetUserFromMemoryOrFallback(client.userID!);
```

Comment:
just to be clear this just refactors the code right? Do we also want to loadHeroUsers in getLocalizedDisplayname? or is that expected to be done by clients?

Reply:
This is still expected to be done in clients. getLocalizedDisplayname has always returned the name sync and not async so we cannot load hero users here

Reply:
Can you add a comment so that sdk consumers know about this then? Thank you :D

Reply:
Added